src/python_routing_v02/troute/routing/compute.py

```diff
@@ -917,6 +929,8 @@ def compute_diffusive_routing(
             nts,
             dt,
             waterbodies_df,
+            comid_bathy,
+            inland_bathyNC,
```

Comment:
These are the critical inputs, don't create these variables in compute.py, rather create them higher up in the workflow and pass then down. This is how we handle all of the other inputs for MC and diffusive routing.

Reply:
Why don't we create these two arguments in def nwm_network_preprocess in preprocess.py where diffusive_network_data is created?   Also, how about creating topobathy_parameter for showing a path to eHydro data under network_topology_parameter in yaml file?